### PR TITLE
Init repo.

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        java: ["8", "15", "16"]
```
Comment (marcingrzejszczak) [inline .github/workflows/maven.yml]: 8,11 and 16 I guess cause those are LTS afair

File: .settings.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (marcingrzejszczak) [inline .settings.xml]: This file is no longer needed

File: docs/pom.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-openfeign-async-docs</artifactId>
+	<version>0.1.0-SNAPSHOT</version>
```
Comment (marcingrzejszczak) [inline docs/pom.xml:8]: `0.0.1-SNAPSHOT` ?

**Review by ryanjbaxter — APPROVED** [review]

File: docs/pom.xml (hunk 3 above)
Comment (spencergibb) [reply docs/pom.xml:8]: I don't think so. In x.y.z, z is a patch release. I think 0.1.0 is fine to start.

File: spring-cloud-openfeign-async-dependencies/pom.xml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+	<description>Spring Cloud OpenFeign Async Dependencies</description>
+	<properties>
+		<feign.version>11.1</feign.version>
+		<feign-form.version>3.8.0</feign-form.version>
+	</properties>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.github.openfeign</groupId>
+				<artifactId>feign-bom</artifactId>
```
Comment (spencergibb) [inline spring-cloud-openfeign-async-dependencies/pom.xml]: maybe import the s-c-openfeign bom, then you don't have to manage the feign versions.

**Review by spencergibb — CHANGES_REQUESTED** [review]
For the most part, LGTM. A couple of small changes.

File: .settings.xml (hunk 2 above)
Comment (OlgaMaciaszek) [reply .settings.xml]: Ok. Removed.

**Review by spencergibb — APPROVED** [review]

**Pull request merged**